### ISSUE TITLE
Option for the MBS Version of Cloudberry

### ISSUE THREAD

**Issue by mpcjames** [jlesage/docker-cloudberry-backup#18]
Hello, love the project and it works very well inside Docker on Synology units. 

I have a question.... Is there a way to change/build the app using Cloudberry MBS (Managed Backup Service) program, rather than the Backup that's offered on the main Cloudberry website?

Essentially each MBS provider gets their own management console and install file for the various operating systems and the provider then licenses the various installs/accounts. I'd like to be able to link Cloudberry inside Docker to the licenses/app offered by MBS if possible. 

Currently you can't do this with this container as it doesn't give the usual MBS login options when you first setup your backup. 

A bit of info about the MBS offering is here https://www.cloudberrylab.com/managed-backup.aspx

Does that make sense?
Thanks

**Comment by jlesage** [comment]
Hi,
Is this software available for download to everyone?

**Comment by mpcjames** [comment]
Hello, 
It's available to everyone who has a MBS (Managed Backup Service) account. The download is available for Windows, Linux etc and are built (on request when new versions are released) and when they are built they include all of your own branding and a unique download link is published. 
When the backup software is installed on client computers is it not called "Cloudberry Backup", it's shown as "XYZ Backup Name" or whatever you choose in your branding options. 

In short, it's exactly the same product as the retail version, just custom branded and licenses are managed and allocated by the reseller.

**Comment by bigtfromaz** [comment]
Here is the download link.  I think this the same product just rebranded.
It appears to be an open download requiring activation after a 15 day trial.

I opened a ticket with Cloudberry Labd this week on an unrelated matter.
Instead of cloudberry backup for Linux all of their emails are being signed
MSP Backup which is part of a new group of products dubbed MS360.

Not sure about MBS but it sounds like a custom branded build would require
a corresponding docker build for each custom download link.

https://www.cloudberrylab.com/download.aspx?prod=cbbub1214&p=linux

In any event we just saw a new release of Cloudberry backup for Linux. It
remains to be seen if these are really two tracks or if the old product
will be renamed.

**Comment by jlesage** [comment]
Yes CloudBerry Lab changed its name, but @mpcjames is talking about a different product of the company: the managed backup (different from the standalone backup this container is implementing).

**Comment by jlesage** [comment]
> It's available to everyone who has a MBS (Managed Backup Service) account.

Then can this account be created by everyone?  The signup form include a couple of business related fields.  I'm asking this question because without account, it would be hard for me to implement and test anything.

**Comment by ptidav** [comment]
> > It's available to everyone who has a MBS (Managed Backup Service) account.
> 
> Then can this account be created by everyone? The signup form include a couple of business related fields. I'm asking this question because without account, it would be hard for me to implement and test anything.

Yes it's possible to create a trial account. Then you can download the package on the linux machine from the download menu.
A container for this app version would be great !
Thanks
David
